### PR TITLE
Convert popover to stylable

### PR DESCRIPTION
**Pull request by amiryonatan**

### CODE REVIEW

File: packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContent.tsx [hunk 1]
```diff
@@ -146,28 +121,29 @@ class DropdownContent extends React.PureComponent<DropdownContentProps, Dropdown
   }
 
   render() {
-    const {selectedIds, classes, fixedHeader, fixedFooter, options, maxHeight} = this.props;
+    const {selectedIds, fixedHeader, fixedFooter, options, maxHeight, className} = this.props;
     const {hoveredIndex} = this.state;
 
     return (
       <div
+        {...style('root ' + className, {}, this.props)}
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContent.tsx]: no need for classname

File: packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContentStyle.st.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+.root::optionDisabled {
+  background-color: value(optionDisabledBackgroundColor);
+}
+
+.root::optionSelected {
+  background-color: value(optionSelectedBackgroundColor);
+}
+
+.root::optionSelected::optionHover {
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContentStyle.st.css]: Maybe .root::optionSelected.root:::optionHover {

File: packages/wix-ui-core/src/components/InputWithOptions/InputWithOptions.st.css [hunk 3]
```diff
@@ -0,0 +1 @@
+.root {}
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/InputWithOptions/InputWithOptions.st.css]: maybe you don't need it

File: packages/wix-ui-core/src/components/Tooltip/Tooltip.st.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  borderWidth: 1px;
+  borderStyle: solid;
+  borderColor: black;
+  borderRadius: 10px;
+  padding: 5px;
+  arrowSize: 5px;
+}
+
+.root {
+  -st-extends: BasePopover;
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Tooltip/Tooltip.st.css]: remove

File: packages/wix-ui-core/src/components/Tooltip/Tooltip.tsx [hunk 5]
```diff
@@ -27,13 +20,6 @@ export class Tooltip extends React.PureComponent<TooltipProps, TooltipState> {
     placement: 'top'
   };
 
-  static propTypes = {
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Tooltip/Tooltip.tsx]: don't remove

File: packages/wix-ui-core/stories/index.tsx [hunk 6]
```diff
@@ -22,30 +22,30 @@ const dropdownOptions =
...
       options={dropdownOptions}/>
   ))
   .add('InputWithOptions Multi select', () => (
     <InputWithOptions
       closeOnSelect={false}
-      dataHook="story-input-with-options-multi"
+      data-hook="story-input-with-options-multi"
       options={dropdownOptions}/>
   ))
   .add('Input', () => (
```
Comment (lbelinsk) [inline packages/wix-ui-core/stories/index.tsx]: only your refactored omponents

File: packages/wix-ui-core/src/baseComponents/Dropdown/Dropdown.tsx [hunk 7]
```diff
@@ -1,24 +1,19 @@
...
 
-export type DropdownClasses = {
-};
-
 export interface TriggerElementProps {
   onKeyDown(evt: React.KeyboardEvent<HTMLElement>);
 }
 
 export interface DropdownProps {
   placement: Placement;
```
[the remark this answers is not in the document]
Comment (amiryonatan) [reply packages/wix-ui-core/src/baseComponents/Dropdown/Dropdown.tsx]: 👍

File: packages/wix-ui-core/src/baseComponents/Dropdown/DropdownStyle.st.css [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  -st-from: "../Popover/PopoverStyle.st.css";
+  -st-default: Popover;
+}
+
+:import {
+  -st-from: "../DropdownContent/DropdownContentStyle.st.css";
+  -st-default: DropdownContent;
+}
+
+:vars {
```
[the remark this answers is not in the document]
Comment (amiryonatan) [reply packages/wix-ui-core/src/baseComponents/Dropdown/DropdownStyle.st.css]: 👍

File: packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContent.tsx (hunk 1 above)
Comment (amiryonatan) [reply packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContent.tsx]: 👍

File: packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContentStyle.st.css (hunk 2 above)
Comment (amiryonatan) [reply packages/wix-ui-core/src/baseComponents/DropdownContent/DropdownContentStyle.st.css]: 👍

File: packages/wix-ui-core/src/components/InputWithOptions/InputWithOptions.st.css (hunk 3 above)
Comment (amiryonatan) [reply packages/wix-ui-core/src/components/InputWithOptions/InputWithOptions.st.css]: 👍

File: packages/wix-ui-core/src/components/Tooltip/Tooltip.st.css (hunk 4 above)
Comment (amiryonatan) [reply packages/wix-ui-core/src/components/Tooltip/Tooltip.st.css]: 👍

File: packages/wix-ui-core/src/components/Tooltip/Tooltip.tsx (hunk 5 above)
Comment (amiryonatan) [reply packages/wix-ui-core/src/components/Tooltip/Tooltip.tsx]: 👍

File: packages/wix-ui-core/stories/index.tsx (hunk 6 above)
Comment (amiryonatan) [reply packages/wix-ui-core/stories/index.tsx]: 👍

**Pull request merged**